src/offset/fixed.rs

```diff
@@ -169,75 +165,6 @@ impl arbitrary::Arbitrary<'_> for FixedOffset {
     }
 }
 
-// addition or subtraction of FixedOffset to/from Timelike values is the same as
```

Comment:
Please don't move code and change it in the same commit, this makes it very hard to review.

Reply:
I have split it up into multiple commits, including the parts from your comment in https://github.com/chronotope/chrono/pull/1071#discussion_r1198850204.